Sources/Card/CardClient.swift

```diff
@@ -123,3 +111,19 @@ public class CardClient {
         delegate?.cardDidCancel(self)
     }
 }
+
+// MARK: - ASWebAuthenticationPresentationContextProviding conformance
+
+extension CardClient: ASWebAuthenticationPresentationContextProviding {
+
+    public func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
```

Comment:
Should we provide merchants with a way to pass their own window in the top level class if they want? I think using the first window _should_ cover most merchants but but may not always be the case. In the clients we could have something like we do on BT `public var activeWindow: UIWindow?` to allow merchants to pass in their own optional `UIWindow` to use instead. Then in here we can check if it's nil and if not return the merchant specified window. Thoughts?

Reply:
Great question! 

So I actually made a [ticket](https://paypal.atlassian.net/browse/DTNOR-749) and a TODO to add that functionality after this PR. But after thinking about it more I ended up closing it and leaving a note why it was a "won't do".

SPLUNKing BT logs shows that 98% of all events are on iPhones vs iPads (multitasking is only supporting on iPads). So seeing as we don't get much traffic at all there combined with the fact that I think supporting multi-tasking for a checkout flow would be pretty rare for a merchant, I figured we could leave this as "won't do", until/if a merchant asks us to explicitly support it. 

On the BT side, we added it in for completeness, not based on merchant request. I wonder if we removed it, if anyone would notice 🤷 

What do you think?

Reply:
Oh dang - that makes a ton of sense, that's for sharing your findings on that! I had assumed it was added because merchants had asked for it previously. I agree in that case that we should not add it for now to clean up the integration path overall and evaluate if merchants ask for expect support! 🚀

Reply:
I dig it! Yeah that would be the real test to see if merchants are actually using it. I think we should try to remove it and see if we get any feedback on it. Based on the data you found, this likely would allow us to really clean up the integration pattern with hopefully no merchant issues.

Reply:
Would that mean removing these [lines of code](https://github.com/braintree/braintree_ios/blob/a447ec02ee5f52940f2ab9c53fdd3dbf0f645c31/Sources/BraintreePayPal/BTPayPalClient.swift#L218-L228) and removing `activeWindow` from `BTPayPalRequest`?

Reply:
Just this [part](https://github.com/braintree/braintree_ios/blob/a447ec02ee5f52940f2ab9c53fdd3dbf0f645c31/Sources/BraintreePayPal/BTPayPalClient.swift#L218-L220), but yes